Description: Use canned version of example that segfaults on armhf.
Author: Doug Torrance <dtorrance@piedmont.edu>
Bug: https://github.com/Macaulay2/M2/issues/1676
Last-Update: 2020-12-09

--- a/M2/Macaulay2/packages/MultiplicitySequence.m2
+++ b/M2/Macaulay2/packages/MultiplicitySequence.m2
@@ -493,11 +493,27 @@
         Text
             Given a monomial ideal I, this function computes the j-multiplicity of I
 	    following the method of Jeffries-Montaño.
-        Example
-            R = QQ[x,y]
-	    I = (ideal"xy5,x2y3,x3y2")^4
-            elapsedTime monjMult I
-            elapsedTime jMult I
+        CannedExample
+            i1 : R = QQ[x,y]
+
+            o1 = R
+
+            o1 : PolynomialRing
+
+            i2 : I = (ideal"xy5,x2y3,x3y2")^4
+
+                         4 20   5 18   6 17   6 16   7 15   8 14   7 14   8 13   9 12   10 11   8 12   9 11   10 10   11 9   12 8
+            o2 = ideal (x y  , x y  , x y  , x y  , x y  , x y  , x y  , x y  , x y  , x  y  , x y  , x y  , x  y  , x  y , x  y )
+
+            o2 : Ideal of R
+
+            i3 : elapsedTime monjMult I
+             -- 0.0945551 seconds elapsed
+
+            o3 = 192
+
+            i4 : elapsedTime jMult I
+             -- 1.15819 seconds elapsed
     SeeAlso
         multiplicitySequence
 	jMult
